### PR TITLE
Add new Css questions

### PR DESCRIPTION
**Pull request by Shaik-mohd-huzaifa**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

File: src/data/css-quiz.ts [hunk 1]
```diff
@@ -1241,6 +1241,16 @@ const cssQuiz = [
     Explanation:
       "You can use border-style: groove; to specify groove border styles around HTML elements.",
     Link: "https://www.freecodecamp.org/news/css-border-style-and-html-code-examples/"
+  },
+  {
+    Question: "In CSS, What will this pseudo class selector do? :nth-child(2)",
```
Comment (jdwilkin4) [inline src/data/css-quiz.ts:1246]:
```suggestion
    Question: "In CSS, what will the following pseudo class selector do :nth-child(2)?",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by Shaik-mohd-huzaifa** [comment]
Changes Applied

**Review by jdwilkin4 — APPROVED** [review]
Congrats on another contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**